mne/io/cnt/cnt.py

```diff
@@ -97,9 +97,9 @@ def _get_cnt_info(input_fname, eog, ecg, emg, misc):
         patient_id = read_str(fid, 20)
         patient_id = int(patient_id) if patient_id.isdigit() else 0
         fid.seek(121)
-        patient_name = read_str(fid, 20)
-        last_name = patient_name[0]
-        first_name = patient_name[-1]
+        patient_name = read_str(fid, 20).split()
+        last_name = patient_name[0] if len(patient_name) > 0 else 'Unknown'
+        first_name = patient_name[-1] if len(patient_name) > 0 else 'Unknown'
```

Comment:
Well... Do we want to have empty fields on purpose? If we save the data as fif and then read it again, it would have Nones as names.